### ISSUE TITLE
Servant shows recommended likes as being original posts

### ISSUE THREAD

**Issue by ThePsionic** [new-xkit/XKit#1152]
![screenshot 2016-09-19 08 46 26](https://cloud.githubusercontent.com/assets/842080/18624183/cc333d26-7e45-11e6-86ba-ff0e64ae0c52.png)
Filter is "If post is original, colorize green"

**Comment by nightpool** [comment]
They all are original posts? I mean, its not showing you reblogs, its showing you original posts...

**Comment by nightpool** [comment]
Also that's a cool servant rule I should use it.

**Comment by ThePsionic** [comment]
@nightpool fair enough. Some of my favorites also include "if post is mine, colorize yellow" and "if post is liked by me, colorize red".

**Comment by nightpool** [comment]
down to re-open this as a papercut if you're still feeling pain on this

**Comment by ThePsionic** [comment]
I always feel pain, so why not
